qos-service-impl/src/main/java/com/palantir/atlasdb/qos/config/QosClientLimitsConfig.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.qos.config;
+
+import java.io.Serializable;
+
+import org.immutables.value.Value;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableQosClientLimitsConfig.class)
+@JsonSerialize(as = ImmutableQosClientLimitsConfig.class)
+public abstract class QosClientLimitsConfig implements Serializable {
+    // These numbers comes from the metrics released and are scaled by a factor of 2,
+    // we should probably scale up more or set to no limit?
+    public static final long BYTES_READ_PER_SECOND_PER_CLIENT = 500_000L;
+    public static final long BYTES_WRITTEN_PER_SECOND_PER_CLIENT = 1_000_000L;
```

Comment:
yup, but the limit depends on the name of the client. I think we should divide this by the number of nodes. How will we discover the node count for each client - maybe add that to the config and discover as `{{client-api.uris}}.count`?

Reply:
The current defaults come from the readBytes and writeBytes metrics collected from the atlasDB clients. Happy to tweak them to more sensible values.

Reply:
I think these are fine for now or at least can be tweaked later after some testing. 

Splitting based on number of nodes will be tricky, since we expect it to be non-static going forward and potentially changing fairly often. If we do something here, I think it can wait, since clients tracking the number of other clients in the same service is a bit annoying. This should be a reasonable version until we get some signal that this approach isn't comprehensive enough.